ament_cmake_pclint/package.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>ament_cmake_pclint</name>
+  <version>0.0.3</version>
+  <description>
+    The CMake API for ament_pclint to perform static code analysis on C/C++
+    code using PCLint.
+  </description>
+  <maintainer email="jp.samper@apex.ai">Juan Pablo Samper</maintainer>
+  <license>Apex.AI Proprietary License</license>
```

Comment:
From this patch it is unclear what this license is. I think each package needs at least a `LICENSE` file.

Reply:
@dirk-thomas It will probably become Apache 2.0 (or another well-known license), I just copy-pasted the code for now, so you could start looking at the content.